server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -318,7 +333,7 @@ export class DocumentStorage implements IDocumentStorage {
 
             // Setting an empty string to deli and scribe denotes that the checkpoints should be loaded from summary.
             const value = inSummary ?
-                await this.createObject(collection, tenantId, documentId, "", "") :
+                await this.createObject(collection, tenantId, documentId, "", "", undefined) :
```

Comment:
`session` is an optional parameter in `createObject()`. So could you omit the `undefined` here?

If not, then don't you also have to add it to the `createObject()` call in line 337 below?

Reply:
Deleted undefined. :)